Apps/Src/RTOSPedals.c

```diff
@@ -0,0 +1,66 @@
+/* Copyright (c) 2020 UT Longhorn Racing Solar */
+
+#include "RTOSPedals.h"
+
+//Accelerator and Brake Percentage
+int8_t accelerator_percentage;
+int8_t brake_percentage;
+
+/**
+* @brief   Initialize both pedals to 0%
+* @param   None
+* @return  None
+*/ 
+static void Pedals_Values_Init(void){
+    accelerator_percentage = 0;
+    brake_percentage = 0;
+}
```

Comment:
I don't think you'd need a function to initialize this. Since the variables are global, they'll be initialized to 0 by default